### PR TITLE
Fixed placeholder when using responsive lazy load

### PR DESCRIPTION
**Pull request by strausr**
The purpose of this PR is to set the width for the placeholder when responsive is used so that the placeholder is not set with width auto and rather takes the width from the window resize. 

The transformation for predominant-color is changed to take the current width rather than the original so that the responsive width is used

### CODE REVIEW

File: projects/angular-cld/src/lib/constants.ts [hunk 1]
```diff
@@ -11,9 +11,10 @@ export const predominantColorTransformPxl = [
   {fetch_format: 'auto', quality: 'auto'}];
 
 export const predominantColorTransform  = [
+  {variables: [['$currWidth', 'w'], ['$currHeight', 'h']]},
```
Comment (asisayag2) [inline projects/angular-cld/src/lib/constants.ts]: @strausr , what are 'h' & 'w'?

File: projects/angular-cld/src/lib/cloudinary-image.component.ts [hunk 2]
```diff
@@ -148,12 +148,17 @@ export class CloudinaryImage
...
   }
 
-  placeholderHandler(options) {
+  placeholderHandler(options, image) {
     const placeholderOptions = {};
-
     Object.keys(options).forEach(name => {
-      placeholderOptions[name] = (name === 'width' && !options[name].startsWith('auto') || name === 'height') ? Math.ceil(parseInt(options[name], 10) * 0.1) : options[name];
-    });
+      if (name === 'width' && options.responsive) {
```
Comment (asisayag2) [inline projects/angular-cld/src/lib/cloudinary-image.component.ts]: @strausr , not following the logic here. 
Please simplify or add some comments to explain what you're trying to achieve. 

Question - if option.responsive is false, is it true to say that placeholderOptions === options?

File: projects/angular-cld/karma.conf.js [hunk 3]
```diff
@@ -25,7 +26,7 @@ module.exports = function (config) {
     colors: true,
     logLevel: config.LOG_INFO,
     autoWatch: true,
-    browsers: ['Chrome'],
+    browsers: ['Chrome', 'Firefox'],
```
Comment (asisayag2) [inline projects/angular-cld/karma.conf.js]: @strausr , is that related to this fix?

File: projects/angular-cld/src/lib/constants.ts (hunk 1 above)
Comment (strausr) [reply projects/angular-cld/src/lib/constants.ts]: This is how you provide user variables.. height and width.

File: projects/angular-cld/src/lib/cloudinary-image.component.ts (hunk 2 above)
Comment (strausr) [reply projects/angular-cld/src/lib/cloudinary-image.component.ts]: Added comment with the explanation. `Question - if option.responsive is false, is it true to say that placeholderOptions === options?` Not exactly.. we want to make use of dimension if they are provided

File: projects/angular-cld/karma.conf.js (hunk 3 above)
Comment (strausr) [reply projects/angular-cld/karma.conf.js]: Yes, we're testing the output in Firefox... The tests require it

Comment (asisayag2) [reply projects/angular-cld/karma.conf.js]: Specifically for this feature?

File: projects/angular-cld/src/lib/cloudinary-image.component.ts (hunk 2 above)
Comment (asisayag2) [reply projects/angular-cld/src/lib/cloudinary-image.component.ts]: I don't get this from the code.  Seems like if responsive==false, nothing happens.

**Comment by asisayag2** [comment]
@strausr , let's have this review f2f.
